lib/openseadragon/engine.rb

```diff
@@ -2,8 +2,8 @@ module Openseadragon
   class Engine < ::Rails::Engine
     isolate_namespace Openseadragon
 
-    config.assets.precompile += %w(openseadragon/*.png)
-
+    initializer 'openseadragon.assets.precompile', after: 'assets' do
```

Comment:
Since you linked to the rails guide on this; does the approach they recommend work?

```
initializer 'openseadragon.assets.precompile' do |app|
  app.config.assets.precompile += %w(openseadragon/*.png)
end
```

(or, the `after: 'assets'` makes me suspect, for no good reason I'm sure)

Reply:
Ah.. I think I just cribbed that from another gem. I've made the change to mirror the rails docs, and tested it.

Reply:
👍